docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin-migration.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Plugin migration guide for v4 - Strapi Developer Docs
+description: Migrate your plugins from v3.6.8 to v4.0 with step-by-step instructions
+sidebarDepth: 2
+canonicalUrl:
+---
+
+<!-- TODO: update SEO -->
+
+# v4 plugin migration guide
+
+The goal of this guide is to get a v3 plugin up and running on v4 as fast as possible by resolving breaking changes. It is not an exhaustive resource for the v4 plugin APIs, which are described in the [Server API](/developer-docs/latest/developer-resources/plugin-api-reference/server.md#server-api-for-plugins) and [Admin Panel API](/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md#admin-panel-api-for-plugins) documentations.
+
+Migrating a plugin from Strapi v3.6.9 to v4.0.x consists in:
+
+- [enabling the plugin](/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/enable-plugin.md)
```

Comment:
I would move this last, enabling should be the final thing you do :P

Reply:
Good point, however, let's just not forget: in technical writing, bullet lists ≠ numbered lists. When you see a bullet list, basically there is no order as each step can be done independently of the other ones.

I'll still update it here and in the TOC as I agree people would probably do it this way (even if the blog post put it the other way 😄 )